configs/disco/README.md

```diff
@@ -0,0 +1,149 @@
+# Disco Diffusion(Google Colab)
```

Comment:
why do you need `(Google Colab)`. A little confusing..

Reply:
For common case, we will write conference name here. However, this is a colab notebook instead, maybe we can remove this. What's your opinion?